cmd/koordlet/main.go

```diff
@@ -38,6 +38,7 @@ import (
 func main() {
 	cfg := config.NewConfiguration()
 	cfg.InitFlags(flag.CommandLine)
+	klog.InitFlags(flag.CommandLine)
```

Comment:
duplicate with the imported initialization in `k8s.io/component-base/logs`

Reply:
fixed. But `k8s.io/component-base/logs` no longer registers flags of klog to flag.CommandLine in init, manual registration is required.

Reply:
ok